config.json

```diff
@@ -34,6 +34,10 @@
       "institution": "OHSU"
     }
   ],
+  "compatibility": {
+      "redcap-version-min": "8.5.4",
```

Comment:
Should this be 8.6.0 because of the bug fix for permissions?

Reply:
Rob mentioned it was going to be added in 8.5.4 and 8.6.0. https://community.projectredcap.org/questions/44112/problem-setting-external-module-user-rights-for-us.html